quicklogic/common/cmake/quicklogic_qlf_arch.cmake

```diff
@@ -113,6 +113,9 @@ function(QUICKLOGIC_DEFINE_QLF_ARCH)
     USE_FASM
 
     ROUTE_CHAN_WIDTH ${QUICKLOGIC_DEFINE_QLF_ARCH_ROUTE_CHAN_WIDTH}
+
+    FIXUP_POST_SYNTHESIS_EXTRA_ARGS
+      --split-ports
```

Comment:
I wonder whether the split ports should be enabled by default and avoid adding an extra argument list.

Reply:
Hmmm, I've been thinking about that too. But ultimately the whole thing is just a workaround for Icarus. Not all SymbiFlow architectures may need it.